dialog/en-us/question_verbs.list

```diff
@@ -0,0 +1,8 @@
+ is
+s
+ are
+re
```

Comment:
what do the "s" and the "re" represent?

Reply:
I do not know but I added it to the ticket.